src/aioswitcher/device/__init__.py

```diff
@@ -121,6 +121,31 @@ def value(self) -> str:
         return self._value  # type: ignore
 
 
+@final
+class LightState(Enum):
```

Comment:
Is this identical to DeviceState?

Reply:
Yes and no..
DeviceState is meant to show the device's state..
But let's say Runner S11 which Runner is the device but it has 2 different modules in it which is 2 lights.
I think it makes more sense to have LightState for each of them because they are not devices inside the device..

Reply:
IDK... It feels weird two identical classes just for their names.
@thecode What do you think?

Reply:
I think we can start without it and if we face a problem we can add it later.

Reply:
Removed and changed to DeviceState..